pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -303,19 +304,19 @@ private void testNewAddedColumns() throws Exception {
     response = postQuery(pqlQuery);
     groupByResult =
         response.getJSONArray("aggregationResults").getJSONObject(0).getJSONArray("groupByResult").getJSONObject(0);
-    Assert.assertEquals(groupByResult.getInt("value"), 0);
+    Assert.assertEquals(groupByResult.getString("value"), "0.00000");
```

Comment:
Let's use getDouble(), then convert to int or long

Reply:
good catch. I was testing this and forgot to change.